spring-boot-project/spring-boot-tools/spring-boot-maven-plugin/src/docs/asciidoc/packaging-oci-image.adoc

```diff
@@ -190,7 +190,7 @@ The value supplied will be passed unvalidated to Docker when creating the builde
 | `false`
 
 | `tags`
-| One or more additional tags to apply to the generated image.
+|One or more additional tags to apply to the generated image. The values provided to the `tags` option should be full image references in the form of `[image name]:[tag]` or `[repository]/[image name]:[tag]`. The name `tags` was chosen to be consistent with the https://buildpacks.io/docs/tools/pack/#pack-cli[`pack CLI`].
```

Comment:
We can remove the last sentence `The name 'tags' was chosen to be consistent with the https://buildpacks.io/docs/tools/pack/#pack-cli['pack CLI']`. While accurate, the statement is true of many arguments here, and we don't need to call this one out specifically.